common/assets/styles/etherwallet-fonts.less

```diff
@@ -1,24 +1,81 @@
 @font-face {
   font-family: 'Lato';
   src: url('../fonts/Lato-Light.woff') format('woff'),
-  url('../fonts/Lato-Light.ttf') format('truetype');
+    url('../fonts/Lato-Light.ttf') format('truetype');
   font-style: normal;
   font-weight: 300;
 }
 
 @font-face {
   font-family: 'Lato';
   src: url('../fonts/Lato-Regular.woff') format('woff'),
-  url('../fonts/Lato-Regular.ttf') format('truetype');
+    url('../fonts/Lato-Regular.ttf') format('truetype');
   font-style: normal;
   font-weight: 400;
 }
 
 @font-face {
   font-family: 'Lato';
   src: url('../fonts/Lato-Bold.woff') format('woff'),
-  url('../fonts/Lato-Bold.ttf') format('truetype');
+    url('../fonts/Lato-Bold.ttf') format('truetype');
 
   font-style: normal;
   font-weight: 700;
 }
+
+@font-face {
+  font-family: 'social-media';
+  src: url('../fonts/social-media.eot');
+  src: url('../fonts/social-media.eot') format('embedded-opentype'),
+    url('../fonts/social-media.woff2') format('woff2'),
+    url('../fonts/social-media.woff') format('woff'),
+    url('../fonts/social-media.ttf') format('truetype'),
+    url('../fonts/social-media.svg') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+.sm-icon {
+  display: inline-block;
+  font: normal normal normal 32px/1 'social-media';
+  text-transform: none;
+  /* Better Font Rendering */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+
+  &.sm-16px {
+    font: normal normal normal 16px/1 'social-media';
+  }
+  &.sm-24px {
+    font: normal normal normal 24px/1 'social-media';
+  }
+  &.sm-32px {
+    font: normal normal normal 32px/1 'social-media';
+  }
+  &.sm-48px {
+    font: normal normal normal 48px/1 'social-media';
+  }
+  &.sm-logo-facebook:before {
+    content: '\ea02';
+  }
+  &.sm-logo-reddit:before {
+    content: '\ea03';
+  }
+  &.sm-logo-github:before {
+    content: '\ea04';
+  }
+  &.sm-logo-twitter:before {
+    content: '\ea05';
+  }
+  &.sm-logo-linkedin:before {
+    content: '\ea06';
+  }
+  &.sm-logo-slack:before {
+    content: '\ea07';
+  }
+  &.sm-logo-medium:before {
+    content: '\ea08';
+  }
+  &.sm-logo-github-2:before {
+    content: '\ea09';
+  }
+}
```

Comment:
I'd move these into their own sass file, with instructions on how to add new ones. I'm not sure where I'd look up the unicode character matches for these.